app/controllers/api/v1/upload_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'csv'
```

Comment:
I think we don't need this as Rails autoloaded it already

Reply:
Sadly, it doesn't work if I remove it: `uninitialized constant UploadForm::CSV`